src/cephadm/cephadm

```diff
@@ -2639,10 +2642,11 @@ def command_bootstrap():
                 f.write(ssh_pub)
             logger.info('Wrote public SSH key to to %s' % args.output_pub_ssh_key)
 
-            logger.info('Adding key to root@localhost\'s authorized_keys...')
-            if not os.path.exists('/root/.ssh'):
-                os.mkdir('/root/.ssh', 0o700)
-            auth_keys_file = '/root/.ssh/authorized_keys'
+            logger.info('Adding key to %s@localhost\'s authorized_keys...' % args.ssh_user)
```

Comment:
can you make this a new function? `command_bootstrap` is already big enough.

Reply:
which part? the chunk of code that creates the ssh key files?

Reply:
ha, look like you don't add a bunch of code here. sorry for the noise.